src/main/java/com/teamchallenge/bookti/review/ReviewController.java

```diff
@@ -0,0 +1,223 @@
+        return ResponseEntity.status(HttpStatus.OK).body(response);
+    }
+
+    @Operation(
+            summary = "Leave a review about the user",
+            responses = {
+                    @ApiResponse(
+                            responseCode = "201",
+                            description = "User reviews information found",
+                            content = {
+                                    @Content(
+                                            mediaType = APPLICATION_JSON_VALUE,
+                                            schema = @Schema(implementation = UserReviewResponseDTO.class))
+                            }),
+                    @ApiResponse(
+                            responseCode = "400",
+                            description = "Bad request or Validation failed",
+                            content = {
+                                    @Content(
+                                            mediaType = APPLICATION_JSON_VALUE,
+                                            schema = @Schema(implementation = ErrorResponse.class))
+                            }),
+                    @ApiResponse(
+                            responseCode = "401",
+                            description = "UNAUTHORIZED",
+                            content = {
+                                    @Content(
+                                            mediaType = APPLICATION_JSON_VALUE,
+                                            schema = @Schema(implementation = ErrorResponse.class))
+                            }),
+                    @ApiResponse(
+                            responseCode = "403",
+                            description = "ACCESS_DENIED",
+                            content = {
+                                    @Content(
+                                            mediaType = APPLICATION_JSON_VALUE,
+                                            schema = @Schema(implementation = ErrorResponse.class))
+                            }),
+                    @ApiResponse(
+                            responseCode = "404",
+                            description = "NOT_FOUND",
+                            content = {
+                                    @Content(
+                                            mediaType = APPLICATION_JSON_VALUE,
+                                            schema = @Schema(implementation = ErrorResponse.class))
+                            }),
+                    @ApiResponse(
+                            responseCode = "500",
+                            description = "INTERNAL_SERVER_ERROR",
+                            content = {
+                                    @Content(
+                                            mediaType = APPLICATION_JSON_VALUE,
+                                            schema = @Schema(implementation = ErrorResponse.class))
+                            })
+            })
+    @PostMapping("/user")
```

Comment:
```suggestion
    @PostMapping
```
You work with review resources, add user data if it needed to payload or reqParam

Reply:
Виправив